topics/admin/tutorials/interactive-tools/slides.html

```diff
@@ -1,13 +1,16 @@
 ---
 layout: tutorial_slides
 logo: "GTN"
+video: true
 
 title: "Galaxy Interactive Tools"
 objectives:
   - Learn the differences between Galaxy Interactive Environments and Galaxy Interactive Tools
   - Have an understanding of what Galaxy Interactive Tools are and how they work
 contributors:
   - natefoo
+  - eancelet
```

Comment:
I don't see you in CONTRIBUTORS.yaml, if you add yourself the build error should go away.

Reply:
:+1: